packages/office-addin-usage-data/.eslintrc.json

```diff
@@ -1,8 +0,0 @@
-{
-  "plugins": [
-    "office-addins"
-  ],
-  "extends": [
-    "plugin:office-addins/recommended"
-  ]
-}
```

Comment:
What linting is left after removing the office plugin?  We should make sure we are only removing the plug-in and not the rest.

Reply:
Sure, I added the lint functionalities back again. Tested them with some examples and ran `npm run lint` on them to check that they are working.